vllm/engine/arg_utils.py

```diff
@@ -1480,7 +1480,9 @@ def _is_v1_supported_oracle(self, model_config: ModelConfig) -> bool:
             return False
 
         # Only support Xgrammar for guided decoding so far.
-        SUPPORTED_GUIDED_DECODING = ["xgrammar", "xgrammar:nofallback"]
+        SUPPORTED_GUIDED_DECODING = [
```

Comment:
didn't we have a special parsing for these on the request?
ig this also works, and it is pretty simple ;)

Reply:
Yeah, but it was checked this way in some places.

I figure we can refactor further if we add more options. I was just trying to fix this quickly since it's a regression from V0.